Tests/Marketplace/search_and_install_packs.py

```diff
@@ -124,102 +108,51 @@ def create_dependencies_data_structure(response_data: dict, dependants_ids: list
         create_dependencies_data_structure(response_data, next_call_dependants_ids, dependencies_data, checked_packs)
 
 
-def get_pack_dependencies(client: demisto_client, pack_data: dict, lock: Lock):
-    """ Get the pack's required dependencies.
+def get_pack_dependencies(client: demisto_client, pack_id: str, lock: Lock) -> dict | None:
+    """
+    Get the pack's required dependencies.
 
     Args:
         client (demisto_client): The configured client to use.
-        pack_data (dict): Contains the pack ID and version.
+        pack_id (str): ID of the pack to get dependencies for.
         lock (Lock): A lock object.
+
     Returns:
-        (list) The pack's dependencies.
+        dict: API response data for the /search/dependencies endpoint.
     """
     global SUCCESS_FLAG
-    pack_id = pack_data['id']
-    logging.debug(f'Getting dependencies for pack {pack_id}')
+
+    api_endpoint = "/contentpacks/marketplace/search/dependencies"
+    body = [{"id": pack_id}]  # 'version' key can be specified alongside "id". Not including it will fetch the latest version.
+
+    logging.debug(f"Fetching dependencies for pack '{pack_id}'.\n"
+                  f"Sending POST request to {api_endpoint} with body: {json.dumps(body)}")
```

Comment:
```suggestion
    logging.debug(f"Fetching dependencies for pack '{pack_id}'.\n"
                  f"Sending POST request to {api_endpoint} with body: {body}")
```

Reply:
`json.dumps()` is more accurate for debugging purposes, as it's formatted in the actual format that is used in HTTP requests (ex: using `"` instead of `'`), and can be copy & pasted straight to CURL, Postman, etc.

Differences:
```python
test = {"a": "test", "b": [1,2,3], "c": {1: 2}}

str(test)
>> "{'a': 'test', 'b': [1, 2, 3], 'c': {1: 2}}"

json.dumps(test)
>> '{"a": "test", "b": [1, 2, 3], "c": {"1": 2}}'
```